lua/autorun/client/boom_headshot.lua

```diff
@@ -1,34 +1,38 @@
 -- These can be accessed without pointing to the IMaterial by using ! before the material string.
 function CreateSpriteMaterials()
-	local params = {["$translucent"] = "1", ["$vertexcolor"] = "1", ["$vertexalpha"] = "1"}
-	for i=1, 8 do
-		params["$basetexture"] = "Decals/blood"..i
-		CreateMaterial("sprite_bloodspray"..i, "UnlitGeneric", params)
-	end
+    local params = {
+        ["$translucent"] = "1",
+        ["$vertexcolor"] = "1",
+        ["$vertexalpha"] = "1"
+    }
+
+    for i = 1, 8 do
+        params["$basetexture"] = "Decals/blood" .. i
+        CreateMaterial( "sprite_bloodspray" .. i, "UnlitGeneric", params )
+    end
 end
 
-hook.Add("Initialize","CreateBlood.TTT", function()
-	CreateSpriteMaterials()
-end)
+hook.Add( "Initialize", "CreateBlood.TTT", function()
+    CreateSpriteMaterials()
+end )
 
 hook.Remove( "CreateClientsideRagdoll", "FWKZT.Headshot.CreateCRagdolls" )
-hook.Add( "CreateClientsideRagdoll", "FWKZT.Headshot.CreateCRagdolls", function( entity, ragdoll )
-	if GAMEMODE_NAME ~= "sandbox" then return end
-	
-	if entity.WasHeadshotDeath and entity:WasHeadshotDeath() then
-		local Head = ragdoll:LookupBone('valvebiped.bip01_head1')
-		if !Head then return end
-		local Pos = ragdoll:GetBonePosition( Head )
 
-		local RagHead = ragdoll:LookupBone('valvebiped.bip01_head1')
-		if !RagHead then return end
-		ragdoll:ManipulateBoneScale(RagHead, Vector(0,0,0) )
+hook.Add( "CreateClientsideRagdoll", "FWKZT.Headshot.CreateCRagdolls", function( entity, ragdoll )
+    if GAMEMODE_NAME ~= "sandbox" then return end
 
-		local ED = EffectData()
-			ED:SetEntity( entity )
-			ED:SetNormal( -entity:GetForward() )
-			ED:SetScale( entity:EntIndex() )
-			ED:SetOrigin( Pos )
-		util.Effect( 'headshot', ED )
-	end
+    if entity.WasHeadshotDeath and entity:WasHeadshotDeath() then
+        local head = ragdoll:LookupBone( "valvebiped.bip01_head1" )
+        if not head then return end
+        local pos = ragdoll:GetBonePosition( head )
+        local ragHead = ragdoll:LookupBone( "valvebiped.bip01_head1" )
+        if not ragHead then return end
+        ragdoll:ManipulateBoneScale( ragHead, Vector( 0, 0, 0 ) )
+        local eD = EffectData()

```

Comment:
```suggestion
        ragdoll:ManipulateBoneScale( ragHead, Vector( 0, 0, 0 ) )

        local eD = EffectData()
```
I think the newline they had there was good, helps break it up a little bit.

Reply:
Yeah, i used my formatter to do the bulk work, i usually add spaces in when im modifying the code but i'll do a run through it an space it properly